client/Fireport/Assets/Scripts/Game/GameBoardBehaviour.cs

```diff
@@ -0,0 +1,57 @@
+using Model.State;
+using UnityEngine;
+
+namespace Game {
+    public class GameBoardBehaviour : MonoBehaviour {
+
+        public GamePieceFactory Factory;
+        public float GridSizeMultiplier = 1;
+
+        private GameObject board;
+        
+        public void Populate(GameState state) {
```

Comment:
This is the meat and potatoes of this PR.  Given a GameState object, this method will use a GamePieceFactory to construct all of the game pieces and place them into the scene.  It handles setting every piece's ID and will eventually be responsible for setting other important information like the Player.Name or BoardSpace.State.